Content.Server/StationEvents/Metric/AnomalyMetric.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Content.Server.Chemistry.Components.SolutionManager;
+using Content.Server.Chemistry.EntitySystems;
+using Content.Server.Power.Components;
+using Content.Server.StationEvents.Metric.Components;
+using Content.Shared.Anomaly.Components;
+using Content.Shared.chaos;
+using Content.Shared.Doors.Components;
+using Content.Shared.FixedPoint;
+using Content.Shared.Fluids.Components;
+
+namespace Content.Server.StationEvents.Metric;
+
+public sealed class AnomalyMetric : StationMetric<AnomalyMetricComponent>
+{
+    [Dependency] private readonly SolutionContainerSystem _solutionContainerSystem = default!;
+
+    public override ChaosMetrics CalculateChaos(EntityUid metric_uid, AnomalyMetricComponent component, ChaosMetricComponent metric,
+        CalculateChaosEvent args)
+    {
+        FixedPoint2 anomalyChaos = 0.0f;
+
+        // Add up the pain of all the firelocks
+        var anomaly_q = EntityQueryEnumerator<AnomalyComponent>();
```

Comment:
are you using VSC or something, we follow standard C# style, which means camelCase for variable names, not snake_case.

Reply:
It's a common mistake I make. I've programmed too much Python and Rust. I sort them out as I realize.